cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -95,11 +98,22 @@ func NewRunner() *Runner {
 	if err != nil {
 		log.Fatalf("error getting user home directory: %v", err)
 	}
+	containerEngine := os.Getenv(envContainerEngine)
+	if containerEngine == "" {
+		if isDockerAvailable() {
+			containerEngine = containerEngineDocker
+		} else if isPodmanAvailable() {
+			containerEngine = containerEnginePodman
+		} else {
+			log.Fatalf("failed to detect any supported container engine")
+		}
+	}
```

Comment:
Can we clear this up to have less if else

Reply:
https://github.com/mesosphere/konvoy-image-builder/pull/789/commits/f82f0c51f5946d7b814506a8926c060b4c23a1e6